runtime/j9vm/exports.cmake

```diff
@@ -350,3 +350,7 @@ if(NOT JAVA_SPEC_VERSION LESS 14)
 		JVM_GetExtendedNPEMessage
 	)
 endif()
+
+if(JITSERVER_SUPPORT)
+	jvm_add_exports(JITServer_CreateServer)
```

Comment:
The first argument should be the library name: `jvm_add_exports(jvm JITServer_CreateServer)`.

Reply:
Done